marge/merge_request.py

```diff
@@ -32,10 +32,13 @@ def fetch_by_iid(cls, project_id, merge_request_iid, api):
 
     @classmethod
     def fetch_all_open_for_user(cls, project_id, user_id, api):
-        all_merge_request_infos = api.collect_all_pages(GET(
-            '/projects/{project_id}/merge_requests'.format(project_id=project_id),
-            {'state': 'opened', 'order_by': 'created_at', 'sort': 'asc'},
-        ))
+        try:
+            all_merge_request_infos = api.collect_all_pages(GET(
+                '/projects/{project_id}/merge_requests'.format(project_id=project_id),
+                {'state': 'opened', 'order_by': 'created_at', 'sort': 'asc'},
+            ))
+        except gitlab.InternalServerError:
```

Comment:
Could you log a warning when this happens? I think it would be good to know about this, otherwise this could happen constantly but from Marge's point of view nothing would be wrong.